assets/r/evaluate_riskchart.R

```diff
@@ -46,6 +46,7 @@ predicted_numeric <- as.numeric(factor(predicted)) - 1
 actual_numeric <- as.numeric(factor(actual)) - 1
 
 # Combine logical NA checks for both vectors.
+# This will be used to filter out invalid indices.
```

Comment:
Why are indices invalid? How do they become invalid and what does it mean to be invalid?